gdal/doc/source/community/service_providers.rst

```diff
@@ -132,6 +133,12 @@ The steps to add a company to the service providers list are outlined in the :re
    :height: 200 px
    :target: `Spatialys`_
 
+.. _`Faunalia`: https://www.faunalia.eu/
+.. |logo_faunalia| image:: ../../images/logo-faunalia.png
+   :class: img-logos
+   :height: 200 px
```

Comment:
make sure the logo width and height are <= 100 px and update the above statement with either :width: 100 px or :height: 100 px accordingly to the max dimension